lbrynet/core/LBRYMetadata.py

```diff
@@ -126,3 +126,6 @@ def _verify_metadata(self, metadata):
                     assert self.meta_version == self['ver'], "version mismatch"
                 break
         assert metadata == {}, "Unknown metadata keys: %s" % json.dumps(metadata.keys())
+
+    def serialize(self):
+        return json.dumps(self).encode("hex")
```

Comment:
it isn't directly needed, if was while i was figuring out what formats go into _mktx in lbryum for claim id and metadata